features/data-layer-migration.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Data Layer Migration
+
+  Background:
+    Given I haven't clicked the migration button
+    And I have completed the basic setup
+
+  Scenario: I don't have any wallets
+    Then I should not see the Data Layer Migration screen
+
+  Scenario: I do have wallets
+    Given I have the following wallets, skiping the data layer migration:
+    | name   | password  |
+    | Wallet |           |
+    Then I should see the Data Layer Migration screen
+    When I click the migration button
+    Then I should see the initial screen
+    When I refresh the application
```

Comment:
@daniloprates this step should not be necessary. I assume the reaction logic in the stores is not properly set as this layer should automatically be removed once user clicks on the migration button...

Reply:
@nikolaglumac Yes, the store is properly set upon user click in the migration button.

The refresh step only tests if the app starts correctly in this scenario, where the user has wallets and already clicked the button.